supertokens_python/recipe/emailverification/utils.py

```diff
@@ -57,26 +44,26 @@ def __init__(
         self.apis = apis
 
 
+MODE_TYPE = Literal["REQUIRED", "OPTIONAL"]
+
+
 class ParentRecipeEmailVerificationConfig:
+    # TODO: Rename this? ^
```

Comment:
yea.. we should. Since now no other recipe embeds this one.

Reply:
Actually, this class is not really needed anymore.. you can just pass in the individual arguments to the `validate_and_normalise_user_input` function (just like in other recipes)

Reply:
TODO.

Reply:
Removed ParentRecipeEmailVerificationConfig

Done.